tests/model_test.cpp

```
@@ -0,0 +1,211 @@
+#include <future>
+#include <memory>
+#include <numbers>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <optional>
+#include <vector>
+
+#include "utility/image/image.details.hpp"
+#include "utility/math/point.hpp"
+
+using namespace pingpong_tracker::identifier;
```

Clang (14.0.6)

[error] do not use namespace using-directives; use using-declarations instead

(google-build-using-namespace,-warnings-as-errors)